### PR TITLE
Nwkukichin

### PR DESCRIPTION
**Pull request by d97hah**

### CODE REVIEW

File: languoids/tree/leng1261/leng1262/nort2971/md.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 [core]
-name = Enlhet Norte
+name = Enxet Sur
```
Comment (xrotwang) [inline languoids/tree/leng1261/leng1262/nort2971/md.ini:3]: This is a regression. I had fixed the names before. This one is Enlhet Norte!

File: languoids/tree/leng1261/leng1262/sout2989/md.ini [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 [core]
-name = Enxet Sur
+name = Enlhet Norte
```
Comment (xrotwang) [inline languoids/tree/leng1261/leng1262/sout2989/md.ini:3]: See above.

**Review by xrotwang — CHANGES_REQUESTED** [review]
There are also tree new glottocodes being minted unnecessarily: `ocoy1235`, `sanp1265` and `sant1466`. It seems you didnt't start out with a clone that included commit 034bba63eca1ba49c9d66c593881fed5d7c16ca8

**Comment by xrotwang** [comment]
@d97hah `hh:ephonwsoc:Hoenigman:Kanjimei` - used as classification reference for Arafundi - seems to be gone. What should it be replaced with?

**Comment by d97hah** [comment]
My bad again, how about now?

Pada tanggal Jum, 19 Nov 2021 pukul 08.41 Robert Forkel <
***@***.***> menulis:

**Comment by xrotwang** [comment]
Looks promising! :) checks running ...

**Review by xrotwang — APPROVED** [review]

**Pull request merged**